css-flexbox-practice.css

```diff
@@ -0,0 +1,120 @@
+.main {
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+}
+.site {
+    display: flex;
+    flex-direction: column;
+    width: 90%;
+    max-width: 800px;
+}
+
+.navigation {
+    display: flex;
+    justify-content: space-between;
+}
+
+.nav-list {
+    list-style-type: none;
+    display: flex;
+}
+
+.list-item {
+    margin-left: 30px;
+}
+
+.section-1 {
+    display: flex;
+    justify-content: space-between;
+    border: 1px solid black;
+    margin-bottom: 50px;
+}
+
+.section-1-left {
+    display: flex;
+    width: 25%;
+}
+
+.section-1-right {
+    display: flex;
+    flex-direction: column;
+    width: 75%;
+    justify-content: center;
+}
+
+.section-2 {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 50px;
+}
+
+.section-3 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-bottom: 50px;
+}
+
+.box {
+    display: flex;
+    flex-direction: column;
+    border: 1px black solid;
+    align-items: center;
+}
+
+.logo {
+    max-width: 100px;
+}
+
+.btn-2 {
+    margin-top: 30px;
+}
+
+.spacing-left {
+    margin-left: 30px;
+}
+
+footer {
+    display: flex;
+    justify-content: space-between;
+}
+
+.footer-list {
+    display: flex;
+    flex-direction: column;
+}
+
+.footer-links {
+    padding-bottom: 15px;
+    list-style-type: none;
+}
+
+.img {
+    max-width: 25%;
+}
+
+.btn-section-1 {
+    margin-left: 20px;
+}
+
+input {
```

Comment:
Do not style html elements directly.
Style a class that you put on the html element.